### PR TITLE
[RFR] Update README

### PR DESCRIPTION
**Pull request by eduardoboucas**
@HugoGiraudel what do you think?

### CODE REVIEW

**Comment by HugoGiraudel** [comment]
I am not sure whether it's good to have examples in the README since we already have them on the site. Maybe kill this section and add a note regarding examples on the site?

**Comment by eduardoboucas** [comment]
Agree. How does it look now?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+I spent quite some time experimenting with different libraries and mixins available out there, but eventually all of them failed to do everything I needed in an elegant way. Some of them wouldn't let me mix set breakpoints with case-specific values, others wouldn't properly handle the CSS OR operator and most of them had a syntax that I found complicated and unnatural.
+
+**include-media** was the result of that experience and it includes all the features I wish I had found before, whilst maintaining a simplistic and natural syntax.
+
+## How to install
+
+- With Bower: `bower install include-media`
+- Manually: get [this file](https://raw.githubusercontent.com/eduardoboucas/include-media/master/dist/_include-media.scss)
+
+Finally, include the file in your project using an `@import`.
```
Comment (HugoGiraudel) [inline README.md]: *using `@import`* or *using an `@import` statement*.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
 
-[Visit the website](http://include-media.com).
+# include-media
+> Simple, elegant and maintainable media queries in Sass
+
+## What?
+**include-media** is a Sass library for writing CSS media queries in an easy and maintainable way, using a natural and simplistic syntax.
+
+## Why?
+I spent quite some time experimenting with different libraries and mixins available out there, but eventually all of them failed to do everything I needed in an elegant way. Some of them wouldn't let me mix set breakpoints with case-specific values, others wouldn't properly handle the CSS OR operator and most of them had a syntax that I found complicated and unnatural.
```
Comment (HugoGiraudel) [inline README.md]: Missing empty line here.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
 <a href="http://include-media.com">!['At' sign](http://include-media.com/assets/images/logo.png)</a>
 
-[Visit the website](http://include-media.com).
+# include-media
+> Simple, elegant and maintainable media queries in Sass
+
+## What?
+**include-media** is a Sass library for writing CSS media queries in an easy and maintainable way, using a natural and simplistic syntax.
```
Comment (HugoGiraudel) [inline README.md]: Missing empty line here.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,37 @@
...
+## How to install
+
+- With Bower: `bower install include-media`
+- Manually: get [this file](https://raw.githubusercontent.com/eduardoboucas/include-media/master/dist/_include-media.scss)
+
+Finally, include the file in your project using an `@import`.
+
+## How to use
+
+Examples can be found [here](http://include-media.com/#features)
```
Comment (HugoGiraudel) [inline README.md]: Missing full-stop.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,37 @@
...
+
+## Learn more
+- [include-media.com](http://include-media.com)
```
Comment (HugoGiraudel) [inline README.md]: Missing empty line here.

File: README.md [hunk 6]
```diff
@@ -1,3 +1,37 @@
...
+- [include-media: Release the docs!](http://include-media.com/documentation/)
```
Comment (HugoGiraudel) [inline README.md]: I am not sure everybody will get that it's the documentation. :P

File: README.md [hunk 7]
```diff
@@ -1,3 +1,37 @@
...
+- [Approaches to Media Queries in Sass](https://css-tricks.com/approaches-media-queries-sass/) (CSS-Tricks)
+- [Write Simple, Elegant and Maintainable Media Queries with Sass](http://davidwalsh.name/sass-media-query) (David Walsh blog)
+- [Create Simple Inline Media Queries with include-media](http://webdevstudios.com/2015/05/18/create-simple-inline-media-queries-include-media/) (WebDevStudios)
+- [Breakpoints and Tweakpoints in Sass](http://www.sitepoint.com/breakpoints-tweakpoints-sass/) (SitePoint)
+
+## The authors
+We are [Eduardo Bouças](https://twitter.com/eduardoboucas) and [Hugo Giraudel](https://twitter.com/hugogiraudel).
```
Comment (HugoGiraudel) [inline README.md]: Missing empty line here.

File: README.md [hunk 8]
```diff
@@ -1,3 +1,37 @@
...
+
+## We want to hear from you
+We'll be on the lookout for your [issues](https://github.com/eduardoboucas/include-media/issues) and [pull requests](https://github.com/eduardoboucas/include-media/pulls) — but make sure you read [this](https://github.com/eduardoboucas/include-media/blob/master/CONTRIBUTING.md) before submitting any code!
```
Comment (HugoGiraudel) [inline README.md]: Missing empty line here.

**Comment by HugoGiraudel** [comment]
I added line notes. Maybe add a license? As well as LICENSE file?

**Pull request merged**